crates/corro-agent/src/agent/tests.rs

```diff
@@ -1157,7 +1298,7 @@ async fn get_rows(
                 |row| row.get(0),
             )?;
             let mut last = 4;
-            // count will be zero for cleared versions
+            // count will be zero for cleared vertest_processions
```

Comment:
Find and replace? :D

Reply:
😅 Sometimes my thinkpad does some weird coy-paste thing when I use the touchpad. Need to disable it..